demisto_sdk/commands/common/tools.py

```diff
@@ -2773,6 +2773,50 @@ def remove_copy_and_dev_suffixes_from_str(field_name: str) -> str:
     return field_name
 
 
+def get_display_name(file_path, file_data={}) -> str:
```

Comment:
Have we checked this function on the new XSIAM entities as well?

Reply:
It's the same function as it was in the UpdateRN class, so it should behave the same as before.
Or maybe I didn't get it right?

Reply:
You understood correctly but I am asking this since we might have missed this in our original effort and thus it will not work here as well.